src/main/resources/schemas/nhinc/common/NhincCommonEntity.xsd

```diff
@@ -248,9 +248,41 @@
         </xsd:sequence>
     </xsd:complexType>
     <xsd:element name="RespondingGateway_ProvideAndRegisterDocumentSetSecuredResponseRequest" type="tns:RespondingGateway_ProvideAndRegisterDocumentSetSecuredResponseRequestType"/>
+    <xsd:complexType name="RespondingGateway_RegisterDocumentSetRequestType">
+        <xsd:sequence>
+            <xsd:element name="assertion" type="nccommon:AssertionType"/>
+            <xsd:element name="nhinTargetCommunities" type="nccommon:NhinTargetCommunitiesType" minOccurs="0"/>
+            <xsd:element name="RegisterDocumentSetRequest" type="ihe:RegisterDocumentSetRequestType"/>
+            <xsd:element name="url" type="nccommon:UrlInfoType" minOccurs="0" maxOccurs="1"/>
+        </xsd:sequence>
+    </xsd:complexType>
+    <xsd:element name="RespondingGateway_RegisterDocumentSetRequest" type="tns:RespondingGateway_RegisterDocumentSetRequestType"/>
 
+    <xsd:complexType name="RespondingGateway_RegisterDocumentSetSecuredRequestType">
+        <xsd:sequence>
+            <xsd:element name="nhinTargetCommunities" type="nccommon:NhinTargetCommunitiesType" minOccurs="0"/>
+            <xsd:element name="RegisterDocumentSetRequest" type="ihe:RegisterDocumentSetRequestType"/>
+            <xsd:element name="url" type="nccommon:UrlInfoType" minOccurs="0" maxOccurs="1"/>
+        </xsd:sequence>
+    </xsd:complexType>
+    <xsd:element name="RespondingGateway_RegisterDocumentSetSecuredRequest" type="tns:RespondingGateway_RegisterDocumentSetSecuredRequestType"/>
 
+    <xsd:complexType name="RespondingGateway_RegisterDocumentSetResponseRequestType">
+        <xsd:sequence>
```

Comment:
Is this the response Type and not the deferred response request? If yes, can you name it RespondingGateway_RegisterDocumentSetResponseType